ramls/request-move.raml

```diff
@@ -0,0 +1,48 @@
+#%RAML 1.0
+title: Circulation
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: Circulation Business Logic API
+    content: <b>API for moving request from one Item to another</b>
+
+types:
+  requests: !include requests.json
+
+/circulation:
+  /requests:
+    /{requestId}:
+      /move:
+        displayName: Move a Request from one Item to another
+        post:
+          description: Move Request to specified Item
+          body:
+            application/json:
+              type: !include request-move.json
+              example: !include examples/request-move.json
+          responses:
+            200:
+              description: "Request has been moved"
+              body:
+                application/json:
+                  type: !include request.json
+                  example: !include examples/request.json
+            404:
+              description: "Request recourd not found"
+              body:
+                text/plain:
+                  type: error
+                  example: "request record with ID \"f3eb6d07-861a-4b76-9937-2ae12768e5f5\" cannot be found"
+            422:
+              description: "Unable to process request"
+              body:
+                application/json:
+                  type: error
```

Comment:
The error schema needs including in order to refer to it here. I think this can be done by including the below in the types section:
```
error: !include raml-util/schemas/error.schema
```

Reply:
I am fixing now.

Reply:
We could use the validation.raml trait as well so we don't explicitly include the 422. Up to you. I think it still requires declaring error/errors anyway. See circulation.raml for examples.